packit_service/sentry_integration.py

```diff
@@ -77,10 +78,11 @@ def configure_sentry(
 
 
 def send_to_sentry(ex):
-    # so that we don't have to have sentry sdk installed locally
-    import sentry_sdk
+    if ServiceConfig.get_service_config().deployment != Deployment.dev:
```

Comment:
Cannot this be handled in the sentry-setup?

Reply:
Please elaborate.

Reply:
We have a place in code where we setup entry:
- If this "filtering" can be moved there.
- Check if sentry itself cannot handle this "filtering".